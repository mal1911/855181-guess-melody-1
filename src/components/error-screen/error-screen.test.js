import React from 'react';
import renderer from 'react-test-renderer';
import ErrorScreen from './error-screen';
import {settings} from '../../mocks/mocks';

it(`ErrorScreen correctly renders`, () => {
  const tree = renderer
    .create(<ErrorScreen mistakes={settings.errorCount} />).toJSON();
  expect(tree).toMatchSnapshot();
});
