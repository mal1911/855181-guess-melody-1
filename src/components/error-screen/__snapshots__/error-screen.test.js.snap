// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreen correctly renders 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
